WriteHDLUtils.py

```diff
@@ -868,7 +882,8 @@ def writeModuleInst_generic(module, hls_src_dir, f_writeTemplatePars,
                     foundMatch = (argname==portname)
                 else:
                     # Use the provided matching rules
-                    foundMatch = f_matchArgPortNames(argname, portname)
+                    if "table" not in argname:
```

Comment:
Could this be put in the relevant matchArgPortNames instead? The TE/TC is not the only module with function arguments that shouldn't be matched to anything, so instead of making this if-statement very long it might be better to put it in matchArgPortNames_XX and make it return False?

Reply:
Wait a minute, I take half of it back. Since the tables in the TE are inputs to the TE top function, HLS does create ports for those tables and they should therefore be matched to something (that isn't in the scripts yet...)